doc/help_modification.md

```diff
@@ -1,6 +1,36 @@
-Modify an existing help
--------------------------
-Each command might have its own help definition. This definition is composed of various _help items_, which might contain their own definitions.
+# Modify an existing help
+
+### Help items
+Before modifying help, it's good to know which help items are available to use.
+There is a list with currently available items:
+```ruby
+Command.extend_help do |h|
+  # Add a simple text to the help output.
+  # content - String - text to be shown
+  # options - Hash   - options (id:, richtext:)
+  h.text(content, options = {})
+  # Add a notification to the help output (LABEL: content).
+  # content - String - text to be shown
+  # options - Hash   - options (same as #text, label:)
+  h.note(content, options = {})
+  # Add a list of items to the help output.
+  # items   - Array - array of items to be shown. Each item can be an array.
```

Comment:
Can we document also the item options? How about "Each item can be either string or an array of `['Key', 'Value', bold: true/false]`. Item options are optional."